spec/models/issue_monitoring_review_spec.rb

```diff
@@ -26,6 +26,7 @@
 
     it "is not valid without a start date" do
       subject.start_date = nil
+      allow(subject.monitoring_review).to receive(:valid?).and_return(false)
```

Comment:
[praise] I think this is a good way to go about this, and will work with both your method and `ActiveModel::Validations`